tests/p2p/test_p2p.py

```diff
@@ -56,6 +53,43 @@ def _add_single_client():
     return _add_single_client()
 
 
+@pytest.fixture()
+def add_clients(setup_bootstrap, setup_clients):
+    """
+    add_clients returns a function that accepts number of replicas
```

Comment:
please improve documentations here

Reply:
fixed in a8ca3677c850eb09dc69144f4c0ef4fbeef737aa